src/test/java/fi/aalto/cs/intellij/notifications/NotificationsUtilityTest.java

```diff
@@ -0,0 +1,21 @@
+package fi.aalto.cs.intellij.notifications;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+import fi.aalto.cs.intellij.common.Version;
+import org.junit.Test;
+
+public class NotificationsUtilityTest {
```

Comment:
should the class name be more specific to `BetaVersionWarning`?

Reply:
Yep, a good catch. :+1: